### DISCUSSION TITLE
Don't appreciate GitHub straight up lying to me to trick me into buying crap.

### DISCUSSION THREAD

**Discussion by nicholasrolstad** [community/community#122060] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Received an email that says: _**"You have 1 new request from members for Copilot Business."**_

No, no I don't. I have a very small team, and nobody requested anything. This is 100% made up.

**_"1 member of your organization has requested access to Copilot Business since you last reviewed requests from members."_**

That's beyond misleading and unethical, it's a straight-up 100% known lie. What a shame.

**Comment by davecheney** [comment]
@nicholasrolstad i'm very sorry that you experienced this. Please contact support, https://support.github.com/contact, or your sales representative (if you are in regular contact with one) who can escalate this to the engineering teams.
